docs/modules/tiered-storage/pages/capacity_planning.adoc

```diff
@@ -0,0 +1,41 @@
+= Capacity Planning
+
+== Planning with the Memory
+
+For each partition of every map, there exists a HybridLog data structure.
+Moreover, each index defined for these maps has a corresponding HybridLog.
+As the HybridLog includes a mandatory memory component, the configuration of IMaps backed by Tiered Storage dictates the minimum required native memory for the system.
+This minimum memory footprint can be calculated with the following formula:
+
+----
+min_memory = (ts_imap_count + ts_index_count) * partition_count
+             * min_hybridlog_mem
+----
+
+The variables are as follows:
+
+- `ts_imap_count`: The number of IMaps where Tiered Storage is enabled.
+- `ts_index_count`: The sum of all indices defined for the IMaps where Tiered Storage is enabled.
+- `partition_count`: The partition count of the Hazelcast cluster; by default 271.
+- `min_hybridlog_mem`: Constant 4MB.
+
+This means that having Tiered Storage enabled for a single IMap, with no index configured, and with the default partition count, the minimum required memory is `(1 + 0) * 271 * 4MB = 1084MB`.
+
+The Hazelcast cluster members validate, on startup, if the native memory configured for the member is greater than or equal to the minimum required memory as calculated above.
+If not, the cluster fails to start.
+
+##TODO: can/should we give a formula for the hash index too?It depends on the entry count.
+##
+
+NOTE: Since the partitions are distributed in the cluster, a single member is not expected to store data from all partitions.
+However, in undesired conditions such as network splits and member shutdowns, all data may end up in a single member.
+This is the reason why Hazelcast always uses the partition count of the cluster for the above validation.
+
+== Planning with the Disk
```

Comment:
while healing after split, we may need `2 x data-capacity`

Reply:
Addressed in https://github.com/hazelcast/hz-docs/pull/1047/commits/0253a16cd6c5ede529c5f0f74b0e730fe62b5d95